zapisy/apps/theses/templates/theses/form_pdf.html

```diff
@@ -38,8 +38,15 @@ <h2>
     <h3>I. Dane studenta</h3>
     <p>Imię i nazwisko: <strong>{{ first_student.get_full_name }}</strong></p>
     <p>Numer albumu: <strong>{{ first_student.matricula }}</strong></p>
-    <h3>II. Dane promotora</h3>
-    <p>Imię, nazwisko, st. naukowy:  <strong>{{ thesis.advisor }}</strong></p>
+    {% if thesis.supporting_advisor %}
+      <h3>IIa. Dane promotora</h3>
+      <p>Imię, nazwisko, st. naukowy:  <strong>{{ thesis.advisor }}</strong></p>
+      <h3>IIb. Dane promotora wspierającego</h3>
+      <p>Imię, nazwisko, st. naukowy:  <strong>{{ thesis.supporting_advisor }}</strong></p>
```

Comment:
```suggestion
      <p>Imię, nazwisko, st. naukowy: <strong>{{ thesis.supporting_advisor }}</strong></p>
```
Nie musimy propagować dalej zastanego niechlujnego HTMLa :slightly_smiling_face: